test/fixtures/utf8-skip-escape-characters.json

```diff
@@ -0,0 +1,61 @@
+{
+    "charset": "utf-8",
+    "headers": {
+        "Project-Id-Version": "gettext-parser",
+        "Report-Msgid-Bugs-To": "andris@node.ee",
+        "POT-Creation-Date": "2012-05-18 14:28:00+03:00",
+        "PO-Revision-Date": "2012-05-18 14:37+0300",
+        "Last-Translator": "Andris Reinman <andris@kreata.ee>",
+        "Language-Team": "gettext-parser <andris@node.ee>",
+        "Language": "",
+        "Content-Type": "text/plain; charset=utf-8",
+        "Content-Transfer-Encoding": "8bit",
+        "Plural-Forms": "nplurals=2; plural=(n!=1);",
+        "mime-version": "1.0",
+        "x-poedit-language": "Estonian",
+        "x-poedit-country": "ESTONIA",
+        "x-poedit-sourcecharset": "utf-8"
+    },
+    "translations": {
+        "": {
+            "": {
```

Comment:
Can you please also remove these headers?
I know some test fixtures have them, but more recent ones don't. Having short files makes it so much easier to find issues in tests. Thanks!

Reply:
I can't remove these. Causes an error. It is there on all other tests too